code/gazecode.m

```diff
@@ -528,9 +586,90 @@ function gazecode(settings)
             disp('Determining fixations...');
             gv.fmark = fixdetectmovingwindow(gv.datx,gv.daty,gv.datt,gv);
             
+        case 'Tobii Pro Glasses 3'
```

Comment:
deze code is bijna helemaal hetzelfde als voor de G2. gebruik case {'Tobii Pro Glasses 2','Tobii Pro Glasses 3'} en in de case gebruik if strcmp() waar het nodig is (alleen readG2DataFiles/readG3DataFiles denk ik)